skimage/io/collection.py

```diff
@@ -386,6 +349,52 @@ def imread_collection(load_pattern, conserve_memory=True):
 
         See `skimage.io.ImageCollection` for details.
 
+        Parameters
+        ----------
+        load_pattern : str or list
+            Pattern glob or filenames to load. The path can be absolute or
+            relative.  Multiple patterns should be separated by a colon,
+            e.g. ``/tmp/work/*.png:/tmp/other/*.jpg``.  Also see
+            implementation notes below.
+        conserve_memory : bool, optional
+            If True, never keep more than one in memory at a specific
+            time.  Otherwise, images will be cached once they are loaded.
+
+        """
+        return ImageCollection(load_pattern, conserve_memory=conserve_memory,
+                               load_func=imread)
+    return imread_collection
+
+
+class MultiImage(ImageCollection):
+
+    """A class containing all frames from multi-frame images.
+
+    Parameters
+    ----------
+    load_pattern : str or list
+        Pattern glob or filenames to load. The path can be absolute or
+        relative.  Multiple patterns should be separated by `os.pathsep`,
+        e.g. ``/tmp/work/*.png:/tmp/other/*.jpg``.
+    conserve_memory : bool, optional
+        Whether to conserve memory by only caching a single frame. Default is
+        True.
+
+    Other parameters
+    ----------------
+    load_func : callable
+        ``imread`` by default.  See notes below.
+
+    Notes
+    -----
+    If ``conserve_memory=True`` the memory footprint can be reduced, however
+        """Return an `ImageCollection` from files matching the given pattern.
+
+        Note that files are always stored in alphabetical order. Also note that
+        slicing returns a new ImageCollection, *not* a view into the data.
+
+        See `skimage.io.ImageCollection` for details.
+
         Parameters
         ----------
         load_pattern : str or list
```

Comment:
```suggestion

```

Reply:
I need to remove lines down to the next occurrence of 'Notes' (something I didn't see well while editing online).